.gitignore

```diff
@@ -1,27 +1,29 @@
-# Docker project generated files to ignore
-#  if you want to ignore files created by your editor/tools,
-#  please consider a global .gitignore https://help.github.com/articles/ignoring-files
+# If you want to ignore files created by your editor/tools, please consider a
+# [global .gitignore](https://help.github.com/articles/ignoring-files).
+
 *.exe
 *.exe~
 *.gz
 *.orig
-test.main
 .*.swp
 .DS_Store
-# a .bashrc may be added to customize the build environment
+
+# local repository customization
+.direnvrc
 .bashrc
 .editorconfig
-.gopath/
-.go-pkg-cache/
+
+# top-level go.mod is not meant to be checked in
+/go.mod
+# build artifacts
 bundles/
-cli/winresources/**/winres.json
 cli/winresources/**/*.syso
+cli/winresources/**/winres.json
```

Comment:
These probably only need a single asterisk (as far as I can tell, it goes only one directory deep under `winresources`).

Alternatively, we could make it _less_ specific, and use `*.syso` and `winres.json`